### PR TITLE
New Members Of The Eu Goodie

### PR DESCRIPTION
**Pull request by GuillermoUrcera**
## Description of new Instant Answer, or changes
 Lists current members of the EU and future candidates 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/members_of_the_eu

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
Arguably it could be better as a fathead or a full goodie. I think if you search for 'members of the eu' it'd be nice if there was an IA to provide the answer. There's no harm in a cheatsheet covering the query but it'll only fire for 'members of the eu cheatsheet' so .. I'm not sure

File: share/goodie/cheat_sheets/json/members-of-the-eu.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            {
+                "key": "Sweden",
+                "val": "EU member country: since 1 January 1995"
+            },
+            {
+                "key": "United Kingdom",
+                "val": "EU member country: since 1 January 1973"
+            }
+        ],
+        "Joining the EU": [
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/members-of-the-eu.json]: Probably need another section, 'Leaving the EU' for the UK

**Comment by sahildua2305** [comment]
@mintsoft exactly!

@GuillermoUrcera would you mind converting this into a Goodie instead? Don't worry if you don't know how to do so. I can help you with it. 😄

**Comment by GuillermoUrcera** [comment]
@sahildua2305

I'd be happy to do so if you could point me in the right direction as to how to do it.

**Comment by mintsoft** [comment]
@GuillermoUrcera The best example of an IA is on the docs : https://docs.duckduckhack.com/walkthroughs/calculation.html

it covers a goodie that takes input and outputs a simple answer. For this IA it will basically only have `triggers` and produce the same answer each time.

**Comment by mintsoft** [comment]
I'm going to close this for now in deference to a coming goodie IA; if the goodie is achievable or people disagree then please re-open this. If you need any help with a goodie implementation @GuillermoUrcera open a PR with what you have and we'll gladly help you polish it up into a working implementation

**Comment by sahildua2305** [comment]
@GuillermoUrcera additionally, this Goodie will be very similar to [this one](https://github.com/duckduckgo/zeroclickinfo-goodies/tree/master/share/goodie/url_escape_codes). So feel free to refer to this one and submit a Pull Request once you have that you want help with or want to get reviewed.

**Pull request closed without merging**